config/eslintrc_react.js

```diff
@@ -40,6 +41,7 @@ module.exports = {
         'react/no-is-mounted': 2,
         'react/no-multi-comp': 0, // Enable to define a multiple component to a single file.
         'react/no-render-return-value': 2,
+        'react/no-redundant-should-component-update': 0, // we'll reconsider this option when we begin to use `PureComponent`.
```

Comment:
> We have to also reconsider react/prefer-stateless-function if we support PureComponent.

Ah, Yes. I feel it would be better to enable `ignorePureComponents` option for it simply.